public/app/types/explore.ts

```diff
@@ -108,7 +97,12 @@ export interface ExploreItemState {
   /**
    * Log query result to be displayed in the logs result viewer.
    */
-  logsResult: LogsModel | null;
+  logsResult?: LogsModel | null;
```

Comment:
As logs are processed outside of the panel they still need to be here separately. At some point the processing should probably be moved into the panel so that panels can have common interface. The logs panel accesses it by directly getting it from redux state.